test/jdk/jdk/incubator/vector/VectorRuns.java

```diff
@@ -28,6 +28,8 @@
 /**
  * @test
  * @modules jdk.incubator.vector
+ * @run main VectorRuns
+ * @run main/othervm/java.security.policy=empty_security.policy VectorRuns
```

Comment:
I just added a minimal test here, so as to not otherwise disturb other areas. This is sufficient to very the fix, and ensure that it does not reoccur.